samples/WorkerBindingSamples/Blob/BlobInputBindingSamples.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Extensions.Logging;
+
+namespace SampleApp
+{
+    public class BlobInputBindingSamples
+    {
+        private readonly ILogger<BlobInputBindingSamples> _logger;
+
+        public BlobInputBindingSamples(ILogger<BlobInputBindingSamples> logger)
+        {
+            _logger = logger;
+        }
+
+        [Function(nameof(BlobInputClientFunction))]
+        public async Task<HttpResponseData> BlobInputClientFunction(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequestData req,
```

Comment:
Recommend removing `Route = null` and `Connection = "AzureWebJobsStorage"` from these samples to keep the code clean and focused. The storage account will be defaulted to the value you're providing, right?

Reply:
Yup it should be!